02-spatial-data.Rmd

```diff
@@ -8,7 +8,7 @@
 This is the first practical chapter of the book, and therefore it comes with some software requirements.
 <!-- toDo: rl-->
 <!-- should we be that specific regarding the r version?-->
-You need access to a computer with a recent version of R installed (R [4.3.0](https://stat.ethz.ch/pipermail/r-announce/2022/000683.html) or a later version).
+You need access to a computer with a recent version of R installed (R [4.3.2](https://stat.ethz.ch/pipermail/r-announce/2023/000697.html) or a later version).
```

Comment:
That version is a lot later, why would you recommend this version? I see no problem with asking for up-to-date R versions though.